tests/tx/test_indexes.py

```diff
@@ -96,10 +99,11 @@ def test_tx_tips_voided(self):
         tx1.timestamp = int(self.clock.seconds())
         tx1.resolve()
         self.assertTrue(self.manager.propagate_tx(tx1, False))
-        self.assertEqual(
-            {tx.hash for tx in self.manager.tx_storage.indexes.mempool_tips.iter(self.manager.tx_storage)},
-            {tx1.hash}
-        )
+        if self.manager._enable_sync_v2:
```

Comment:
Isn't it better to check if the index itself exists (instead of sync v2)? It seems safer for future changes.

Reply:
Thanks!